collection/source.csv

```diff
@@ -795,29 +831,32 @@ brownfield-land,brownfield-land,local-authority-eng:WOK,0eb8e8e249606cf5295ae92c
-brownfield-land,brownfield-land,local-authority-eng:WYE,000a05de97fae4ea6e8c7ebbb6adf7ed07f595ef5cf322a620503c4a3113fd0d,http://www.wyreforestdc.gov.uk/planning-and-buildings/planning-policy/brownfield-land-register.aspx,,,,
+brownfield-land,brownfield-land,local-authority-eng:WYE,000a05de97fae4ea6e8c7ebbb6adf7ed07f595ef5cf322a620503c4a3113fd0d,http://www.wyreforestdc.gov.uk/planning-and-buildings/planning-policy/brownfield-land-register.aspx,,,,2020-07-08
 brownfield-land,brownfield-land,local-authority-eng:WYE,36c1c91ee6ec46da952246ce66916adf3200f7b14d42f5a0814948177b84dde4,http://www.wyreforestdc.gov.uk/planning-and-buildings/planning-policy/brownfield-land-register.aspx,,,,
 brownfield-land,brownfield-land,local-authority-eng:WYE,80006946b17efadd28ba6d09a5886d01a6c773a5f119ba6872666374e6e5f5a7,http://www.wyreforestdc.gov.uk/planning-and-buildings/planning-policy/brownfield-land-register.aspx,,,,
 brownfield-land,brownfield-land,local-authority-eng:WYO,466796e9f6e18591dead03d9e7836e02d6494ec1a5ac9e079f5cc7e1a7dfdc9e,https://www.wycombe.gov.uk/pages/Planning-and-building-control/Adopted-Wycombe-planning-policy-and-guidance/Brownfield-land-register.aspx,,,2019-06-21,
 brownfield-land,brownfield-land,local-authority-eng:WYO,7b6a413b1c1018215fe400dae82e6366b80fb8f89c02402c7c7abe20cf746da4,https://www.wycombe.gov.uk/pages/Planning-and-building-control/Adopted-Wycombe-planning-policy-and-guidance/Brownfield-land-register.aspx,,,2019-08-21,
 brownfield-land,brownfield-land,local-authority-eng:WYO,daf78166cf6af62a0b40625a2fd6c8401f27b9fffab4a92d9bd3742bcedea4a4,https://www.wycombe.gov.uk/pages/Planning-and-building-control/Adopted-Wycombe-planning-policy-and-guidance/Brownfield-land-register.aspx,,,2019-08-21,
 brownfield-land,brownfield-land,local-authority-eng:WYO,1234a961deb842fca9c1a660ca8d02ae26a8273341df8a490738d4efccf71eb7,https://www.wycombe.gov.uk/pages/Planning-and-building-control/Adopted-Wycombe-planning-policy-and-guidance/Brownfield-land-register.aspx,,,,2019-11-01
-brownfield-land,brownfield-land,local-authority-eng:WYR,05588db376760416f9d698a5ab405f416d84e733f1b932cb899edf3e606748a6,https://www.wyre.gov.uk/info/200317/planning_policy/1194/brownfield_land_register,,,2018-09-01,
-brownfield-land,brownfield-land,local-authority-eng:WYR,f52de3d0d6050f4313917a3cd4e0728a3400a62df501f80f85d3d13ad9afffd8,,,,,2019-11-24
+brownfield-land,brownfield-land,local-authority-eng:WYR,05588db376760416f9d698a5ab405f416d84e733f1b932cb899edf3e606748a6,https://www.wyre.gov.uk/info/200317/planning_policy/1194/brownfield_land_register,,,2018-09-01,2020-10-08
+brownfield-land,brownfield-land,local-authority-eng:WYR,f52de3d0d6050f4313917a3cd4e0728a3400a62df501f80f85d3d13ad9afffd8,,,,2019-11-24,2020-10-08
+brownfield-land,brownfield-land,local-authority-eng:WYR,4f2c2b32a7fb44c2a778354ff3a42688f888f42bd7a344052ea6e494cca0db2a,https://www.wyre.gov.uk/info/200317/planning_policy/1194/brownfield_land_register,,,2020-10-08,

```

Comment:
is that the right start date?

Reply:
https://github.com/digital-land/brownfield-land-collection/commit/874677ae17532bafa6a36942e428d3e42ac68bac